spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/reactor/LazyCurrentTraceContext.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2013-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.sleuth.instrument.reactor;
+
+import brave.propagation.CurrentTraceContext;
+import brave.propagation.TraceContext;
+import org.apache.commons.logging.Log;
```

Comment:
ps what's the latest with logging libraries? I moved this from the other file, but not sure what we should be using. As it surprises people I might make a comment above the imports.

Reply:
Spring still uses commons logging https://github.com/spring-projects/spring-framework/tree/master/spring-jcl/src/main/java/org/apache/commons/logging

Reply:
thanks. I asked someone to review and they stopped surprised about this..  now I know how to answer.  cheers!